ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/filter/FilterManager.java

```diff
@@ -266,6 +266,16 @@ void recordPendingTransactionEvent(final Transaction transaction) {
     return logs;
   }
 
+  public BlockParameter getToBlock(final String filterId) {
+    final LogFilter filter = filterRepository.getFilter(filterId, LogFilter.class).orElse(null);
+    if (filter == null) {
+      return null;
+    } else {
+      filter.resetExpireTime();
```

Comment:
Why do we need to reset the expire time here?

Reply:
removed